src/components/ebay-notice/migrator.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @description
+ * Migrates each notice to different types
+ */
+
+// Types of buttons that if they are direct children of root will be wrapped in footer
+const buttonTypes = ['ebay-button', 'ebay-menu-button'];
+
+function setAttribute(el, oldName, newName) {
+    if (el.hasAttribute(oldName)) {
+        const attribute = el.getAttribute(oldName);
+        attribute.name = newName;
+    }
+}
+
+function migrator(el, context) {
+    const type = el.hasAttribute('type') && el.getAttributeValue('type').value;
```

Comment:
This migrator will only work if the `type` attribute is a string literal.

We should probably error in the case that it's not... Although that would be a breaking change. We can discuss if you like.